ZenSorcere/EW-MovieMeetUp
# Mpg m1 ewmovies setup

This PR addressed agreed upon Milestone 1: Incorporating EW xml api to display Cinema movies list in the Event Creation page, instead of pulling from the PopMovies DynamoDB table listings.

Updates the following files:
- Package.json/Package-lock.json: library additions
- apicall.service.ts: updated to call EW xml api for pulling movie information
- app-routing.module.ts: updated to remove authguard features from event page (for now?)
- app.module.ts: turns of the HTTP interceptor, as the authorization header being automatically attached was causing CORS issues with the lamda server for the xml api call.
- event.component.ts: needed changes to display xml api data
- movies.ts: added interfaces for EW movie items.

Verification steps:

- launch server in terminal from VS Code using npm start.
- once launched, open browse window for 'localhost:4200'.
- enter test login name and password.
- select 'create event' button.

Result:
- Event page should load EW Movie posters and titles.
